docs/content/advanced/backend/entities/index.md

```diff
@@ -1,23 +1,6 @@
 # Create an Entity
-
 In this document, you’ll learn how you can create an [Entity](overview.md).
-
-## Prerequisites
-
-It’s recommended to create a `tsconfig.json` file in the root of your Medusa server with the following content:
-
-```jsx
-{
-  "compilerOptions": {
-    "experimentalDecorators": true
-  }
-}
-```
-
-This will remove any errors that show up in your IDE related to experimental decorators.
-
 ## Create the Entity
-
```

Comment:
Don't remove this empty line. There should be one empty line before and after every heading